src/components/EditableSelectList.vue

```diff
@@ -12,12 +12,12 @@
           :value="selectedObjectType"
         />
         <span v-else>{{ objectTypes[0].display || objectTypes[0].displayName }}</span>
-        <a @click="addRow" v-if="addRow" class="add-new">
+        <a @click="addRow" v-if="addRow" class="add-new" title="Create new">
```

Comment:
this is a function being passed in from the parent component and it can potentially be null, right?

Reply:
right, if it's not passed in it's `undefined`. Sometimes (like with window and daylighting control instances), it's not allowed to create instance from the library.